easybuild/easyconfigs/r/R/R-3.3.3-intel-2016b-X11-20160819.eb

```diff
@@ -0,0 +1,2250 @@
+name = 'R'
+version = '3.3.3'
+x11ver = '20160819'
+versionsuffix = '-X11-%s' % x11ver
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['5ab768053a275084618fb669b4fbaadcc39158998a87e8465323829590bcfc6c']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=yes --enable-R-shlib"
+# Actually use Tcl/Tk
+configopts += ' --with-tcl-config=$EBROOTTCL/lib/tclConfig.sh --with-tk-config=$EBROOTTK/lib/tkConfig.sh '
+# Enable graphics capabilities for plotting.
+configopts += " --with-cairo --with-libpng --with-jpeglib --with-libtiff"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+    ('X11', x11ver),
+    ('Mesa', '12.0.2'),
+    ('libGLU', '9.0.0'),
+    ('cairo', '1.14.6'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('bzip2', '1.0.6'),
+    ('XZ', '5.2.2'),
+    ('zlib', '1.2.8'),
+    ('SQLite', '3.13.0'),
+    ('PCRE', '8.39'),
+    ('libpng', '1.6.24'),  # for plotting in R
+    ('libjpeg-turbo', '1.5.0'),  # for plottting in R
+    ('LibTIFF', '4.0.6'),
+    ('Java', '1.8.0_92', '', True),  # Java bindings are built if Java is found, might as well provide it
+    ('Tcl', '8.6.5'),  # for tcltk
+    ('Tk', '8.6.5'),  # for tcltk
+    ('cURL', '7.49.1'),  # for RCurl
+    ('libxml2', '2.9.4'),  # for XML
+    ('GDAL', '2.1.3', '-Python-2.7.12'),  # for rgdal
+    ('PROJ', '4.9.2'),  # for rgdal
+    ('GMP', '6.1.1'),  # for igraph
+    ('NLopt', '2.4.2'),  # for nloptr
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.2h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+name_tmpl = '%(name)s_%(version)s.tar.gz'
+ext_urls = [
+    'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+    'http://cran.r-project.org/src/contrib/',  # current version of packages
+    'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+]
+
+# !! order of packages is important !!
+# packages updated on March 7th 2017
+exts_list = [
+    'base',
+    'datasets',
+    'graphics',
+    'grDevices',
+    'grid',
+    'methods',
+    'splines',
+    'stats',
+    'stats4',
+    'tools',
+    'utils',
+    ('Rmpi', '0.6-6', {
+        'patches': ['Rmpi-0.6-5_impi5.patch'],
+        'source_tmpl': name_tmpl,
+        'source_urls': ext_urls,
```

Comment:
please use `exts_default_options` like we do in recent R easyconfigs, this is too much copy-pasta...